### PR TITLE
Pull request for Bugfix/login

### LINKED ISSUE #7: Login bugs
This issue is all about bugfix/login.

### PR DESCRIPTION
**Pull request by phyoko72**
This is a pull request for Bugfix/login. Please check it.

### CODE REVIEW

File: aa.txt [hunk 1]
```diff
@@ -5,3 +5,4 @@ changed by feature/change-password
 changed by feature/change-password local branch
 changed by feature/logout local branch
 changed by feature/signin local branch
+Volutpat periculis repudiare ad pri, diam viris conceptam sed te.
```
Comment (phoelapyae23) [inline aa.txt]: add another line of code here.

**Review by phoelapyae23 — CHANGES_REQUESTED** [review]
Pls do these change...

File: aa.txt (hunk 1 above)
Comment (phyoko72) [reply aa.txt]: OK.

**Review by phoelapyae23 — APPROVED** [review]
OK. Go ahead...

**Pull request merged**